l1-contracts/test/foundry/unit/concrete/governance/Governance/ChainAdmin.t.sol

```diff
@@ -0,0 +1,134 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.24;
+
```

Comment:
it seems to me that this file lacks tests for the `_validateCall` function, i.e. no tests for the case when a restriction forbids certain calls